pkg/handlers/generic/mutation/extraapiservercertsans/inject.go

```diff
@@ -64,11 +79,13 @@ func (h *extraAPIServerCertSANsPatchHandler) Mutate(
 	if err != nil {
 		return err
 	}
-	if !found {
-		log.V(5).Info("Extra API server cert SANs variable not defined")
+	if !found && len(defaultAPICertSANs) == 0 {
+		log.V(5).Info("No Extra API server cert SANs needed to be added")
 		return nil
 	}
 
+	extraSans := append(extraAPIServerCertSANsVar, defaultAPICertSANs...)
```

Comment:
nit, the "extraSans" being a combination of "extraAPIServerCertSANsVar" and "defaultAPICertSANs" is a little confusing, wdyt of just `apiServerCertSANsVar`?

```suggestion
	apiServerCertSANsVar := append(extraAPIServerCertSANsVar, defaultAPICertSANs...)
```

Reply:
i like it!